.github/workflows/auto-merge.yaml

```diff
@@ -0,0 +1,42 @@
+name: Auto Rebase and Merge
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+  pull_request_review:
+    types: [submitted]
+  status: {}
+
+jobs:
+  auto-rebase:
+    name: Auto Rebase
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+      
+      - name: Auto Rebase
+        uses: cirrus-actions/rebase@1.8
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This workflow was archived by the owner on May 7, 2024. It's seems that is doing it's work, but if we see something weird we should consider to substitute it. For now it's ok!